src/together/resources/finetune.py

```diff
@@ -22,9 +23,17 @@
     TrainingType,
     FinetuneLRScheduler,
     FinetuneLinearLRSchedulerArgs,
+    FinetuneCheckpoint,
 )
-from together.types.finetune import DownloadCheckpointType
-from together.utils import log_warn_once, normalize_key
+from together.types.finetune import DownloadCheckpointType, FinetuneEventType
+from together.utils import (
+    log_warn_once,
+    normalize_key,
+    format_event_timestamp,
+    get_event_step,
+)
+
+_FT_JOB_REGEX = r"^ft-[\dabcdef-]+:\d+$"
```

Comment:
What should be the behavior if the user passes a FT job ID without a colon? I believe this expression won't match, perhaps it's best to rename it to clarify that it only handles job IDs with step numbers?